test/extensions/filters/http/common/empty_http_filter_config.h

```diff
@@ -44,6 +44,29 @@ class EmptyHttpFilterConfig : public Server::Configuration::NamedHttpFilterConfi
   const std::string name_;
 };
 
+class UpstreamFilterConfig : public virtual Server::Configuration::UpstreamHttpFilterConfigFactory {
+public:
+  virtual Http::FilterFactoryCb
+  createDualFilter(const std::string& stat_prefix,
```

Comment:
Can you chime in on the naming "Dual"? Is this the terminology we're going with for filter config that works for both upstream and downstream filters? It was a bit confusing at first

Reply:
Yea a follow up makes sense. Make we'll get a better sense of a good name for it.